tls/s2n_early_data_io.c

```diff
@@ -38,6 +38,15 @@ int s2n_end_of_early_data_recv(struct s2n_connection *conn)
     return S2N_SUCCESS;
 }
 
+bool s2n_is_rejected_early_data(struct s2n_connection *conn)
+{
+    return conn && (conn->early_data_state == S2N_EARLY_DATA_REJECTED)
+            /* Only servers receive early data. */
+            && (conn->mode == S2N_SERVER)
```

Comment:
Add `POSIX_ENSURE_REF(conn);` here.

Reply:
This method doesn't have a posix return type and the case where conn==NULL is covered.